src/rules/noRequireImportsRule.ts

```diff
@@ -0,0 +1,48 @@
+/*
+ * Copyright 2015 Palantir Technologies, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+export class Rule extends Lint.Rules.AbstractRule {
+    public static FAILURE_STRING = "require() style import is forbidden for module: ";
+
+    public apply(sourceFile: ts.SourceFile): Lint.RuleFailure[] {
+        return this.applyWithWalker(new NoRequireImportsWalker(sourceFile, this.getOptions()));
+    }
+}
+
+class NoRequireImportsWalker extends Lint.RuleWalker {
+
+    public visitVariableStatement(node: ts.VariableStatement) {
+        var declarations : ts.VariableDeclaration[] = node.declarationList.declarations;
+        declarations.forEach(this.processDeclaration.bind(this));
```

Comment:
we generally don't use `Function.prototype.bind` in TS because you lose out on some type checking.

perhaps you could use the new `for-of` syntax here? if not, then just expand this to:

```
declarations.forEach((declaration) => this.processDeclaration(declaration));
```